crates/hir-expand/src/name.rs

```diff
@@ -85,6 +85,11 @@ impl Name {
         Self::new_text(text)
     }
 
+    pub fn new_root(text: &str) -> Name {
+        // The edition doesn't matter for hygiene.
```

Comment:
It technically does matter for our displaying of names (it should affect the r# escaping), but I don't think that is too important here for now, given our Names are not properly using hygiene right now

Reply:
Displaying of names does not depend on the edition they were written with but on the edition were they are shown, and it's already handled.

Reply:
Right